core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -537,41 +536,34 @@ public void start(Boolean isPrimary) throws IOException {
         mUfsManager.addMount(mountInfo.getMountId(), new AlluxioURI(key), ufsConf);
       }
       // Startup Checks and Periodic Threads.
+
       // Rebuild the list of persist jobs (mPersistJobs) and map of pending persist requests
```

Comment:
This comment is no longer accurate?

Reply:
it's still doing the same thing, it just isn't iterating over every single file